### PR TITLE
Added basic attachment support for android

### PR DESCRIPTION
**Pull request by nesvold**
Added basic support for email attachment for android 📬

### CODE REVIEW

**Comment by ricardofbarros** [comment]
@chirag04 can we get this merged, please?

File: android/src/main/java/com/chirag/RNMail/RNMailModule.java [hunk 1]
```diff
@@ -41,13 +42,13 @@ public String getName() {
    */
   private String[] readableArrayToStringArray(ReadableArray r) {
     int length = r.size();
-    String[] recipients = new String[length];
+    String[] arr = new String[length];
```
Comment (chirag04) [inline android/src/main/java/com/chirag/RNMail/RNMailModule.java]: can you not rename this?

**Comment by chirag04** [comment]
One nit otherwise LGTM.

File: android/src/main/java/com/chirag/RNMail/RNMailModule.java (hunk 1 above)
Comment (nesvold) [inline android/src/main/java/com/chirag/RNMail/RNMailModule.java]: sure. done.

Comment (chirag04) [inline android/src/main/java/com/chirag/RNMail/RNMailModule.java]: sorry for not being clear. that not renaming it i meant keep it as `recipients`.

Comment (nesvold) [inline android/src/main/java/com/chirag/RNMail/RNMailModule.java]: ah. ok. check new commit; renamed method to clarify the intent (to convert a recipients readable arr to string array).

**Comment by chirag04** [comment]
Ok. let me just merge this and fix the nits.

**Comment by chirag04** [comment]
sorry i got confused between the naming. I fixed those on master. 

published 2.0 on npm. Let me know how it goes.

thanks 👍

**Pull request merged**